substrate/frame/revive/src/lib.rs

```diff
@@ -82,9 +82,12 @@ use sp_runtime::{
 };
 
 pub use crate::{
-	address::{create1, create2, is_eth_derived, AccountId32Mapper, AddressMapper},
+	address::{
+		create1, create2, is_eth_derived, AccountId32Mapper, AddressMapper, TestAccountMapper,
+	},
 	exec::{MomentOf, Origin},
 	pallet::*,
+	wasm::RuntimeCosts,
```

Comment:
Re-export from `precompile` module. So we have everything needed for pre-compiles in one space.